_includes/payment-url.md

```diff
@@ -1,9 +1,8 @@
 {% assign when = include.when | default: "selecting the payment instrument" %}
 {% assign api_resource = include.api_resource %}
-{% assign payment_order = include.payment_order | default: false %}
 {% assign full_reference = include.full_reference | default: false %}
 
-{% if payment_order %}
+{% if api_resource == "checkout" %}
```

Comment:
`checkout` isn't an API resource. `checkout` needs to be changed to `paymentorders` in the entire file.

```suggestion
{% if api_resource == "paymentorders" %}
```